Gemfile

```
@@ -23,8 +23,8 @@ end
 
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
-  gem "rubocop", "~> 0.82.0"
-  gem "rubocop-rails", "~> 2.5"
+  gem "rubocop", '~> 1.0', '< 1.19'
+  gem "rubocop-rails"
```

Let's not remove the constaint on `rubocop-rails`, intead we could just do:

```suggestion
  gem "rubocop-rails", "~> 2.26"
```